iina/AppDelegate.swift

```diff
@@ -100,8 +146,20 @@ class AppDelegate: NSObject, NSApplicationDelegate {
       parsePendingURL(url)
     }
 
-    // check whether showing the welcome window after 0.1s
-    Timer.scheduledTimer(timeInterval: TimeInterval(0.1), target: self, selector: #selector(self.checkForShowingInitialWindow), userInfo: nil, repeats: false)
+    let _ = PlayerCore.first
```

Comment:
I suggest wrapping this line into a class function of PlayerCore. Kind of wired.